src/components/sidenav/sidenav.js

```diff
@@ -229,6 +229,9 @@ function SidenavDirective($mdMedia, $mdUtil, $mdConstant, $mdTheming, $animate,
     var lastParentOverFlow;
     var triggeringElement = null;
     var promise = $q.when(true);
+    var skipSidenav = false;
+    var skipNextUpdate = false;
+    var $window = angular.element(window);
```

Comment:
You can use DI to get [$window ](https://docs.angularjs.org/api/ng/service/$window)